packages/dashboard/src/components/actions/settings.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { Box, Checkbox, Modal, NonCancelableCustomEvent, SpaceBetween } from '@cloudscape-design/components';
```

Comment:
Nit: You should use default exports on Cloudscape imports to enable treeshaking.

Reply:
I'll add a story to the backlog to sweep through all of these!